source/administration/current-version.rst

```diff
@@ -0,0 +1,42 @@
+Current Version
```

Comment:
Another thought that came to mind is that I think this page should include all versions up until the ESR version. E.g. if we do a security dot release, we normally backport to the current version, two previous versions, and the ESR version. So people on any of the previous versions might be confused about where to get the download links if they don't know about the existence of the archived section.

So my thinking is to include the same versions on this page as on the changelog. What do you think?

Reply:
I guess this is a discussion that involves the complete process. :)

Why is there even a 5.10.1 when 5.11 is already there? As a customer on the **non ESR** branch i am anyway "forced" to be on the current version. When there is a bug it is anyway just fixed with the latest "current branch". And a user with for example 5.10.1 is anyway forced to update with 5.13. So the user gains 3 months.

And 5.10.1 is only promoted by Tweet and Blogpost. On the webpage (i guess the most important download source?) is only the latest version.

I would suggest that Mattermost "just" maintain two branches. "Current branch" and "ESR". No back-porting of security fixes for older "current branch" versions.

Benefits:
- Support needs to cover only the current MM version and ESR
- Clear and easy release schedule (2 versions to maintain)
- Clear communication to customers: Either "current branch" or "ESR"

Reply:
I will discuss this with the team.

Reply:
I've started to discuss this with PMs and one benefit for supporting multiple versions is that lot of our Enterprise customers are not on the current branch because they have different release cycles and hence don't upgrade every month. If they upgrade from 5.9 to 5.13, it’s helpful for them to have quick access to all changes between those versions. I'm continuing with discussing this with PMs and doing research and will have more updates in the coming weeks.

Reply:
Maybe there is a misunderstanding. In the current-version.rst is just the download to the latest "Current Branch" and ESR Version. The changelog with all the changes are still in the changelog.html and the customers can see there all the changes.

Reply:
My note above was mainly referring to the point you outlined regarding maintaining only two release branches (current and ESR).

I will be doing some research on what versions other companies like GitLab normally include in a  changelog and release notes, and that will help me finalize the review of this PR.